libjulia/backends/evm/EVMCodeTransform.cpp

```diff
@@ -351,6 +345,44 @@ void CodeTransform::operator()(FunctionDefinition const& _function)
 	checkStackHeight(&_function);
 }
 
+void CodeTransform::operator()(ForLoop const& _forLoop)
+{
+	Scope* originalScope = m_scope;
+	// We start with visiting the block, but not finalizing it.
+	m_scope = m_info.scopes.at(&_forLoop.pre).get();
+	int stackStartHeight = m_assembly.stackHeight();
+
+	for (auto const& statement: _forLoop.pre.statements)
```

Comment:
Yes, but I think using iterators manually is evil in most cases. I will add a helper.